plugins/k8saudit-eks/rules/k8s_audit_rules.yaml

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright (C) 2022 The Falco Authors.
+#
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+- required_engine_version: 15
+- required_plugin_versions:
+  - name: k8saudit-eks
+    version: 0.1.0
+
+- rule: Dummy rule
+  desc: >
+    Dummy rule
+  condition: >
+    ka.verb in (get,create,delete,update)
+  output: user=%ka.user.name verb=%ka.verb target=%ka.target.name target.namespace=%ka.target.namespace resource=%ka.target.resource
+  priority: WARNING
+  source: k8s_audit
+  tags: [k8s]
```

Comment:
Could we remove the `rules/` folder from here?

(otherwise, the CI will automatically publish a package for that)